akka-docs/src/main/paradox/stream/operators/Source-or-Flow/mapConcat.md

```diff
@@ -15,6 +15,19 @@ Transform each element into zero or more elements that are individually passed d
 ## Description
 
 Transform each element into zero or more elements that are individually passed downstream.
+Returning an empty iterable results in zero elements being passed downstream
+rather than the stream being cancelled.
+
+## Example
+
+The following takes a stream of integers and duplicates each element.
```

Comment:
"duplicates" might be mistaken for an arithmetic operation for numbers. Maybe "emits two elements for each incoming element"?